src/tnsu/simple_update.py

```diff
@@ -79,6 +85,19 @@ def __init__(self, tensor_network: TensorNetwork, j_ij: list, h_k: float, s_i: l
         self.print_process = print_process
         self.converged = False
 
+    @property
+    def tensors(self) -> list[np.ndarray]:
+        return self.tensor_network.tensors
+
+    @property
+    def weights(self) -> list[np.ndarray]:
+        return self.tensor_network.weights
+    
+    @property
+    def structure_matrix(self) -> list[np.ndarray]:
+        return self.tensor_network.structure_matrix
+
+
```

Comment:
I like that! How is that working with the current `SimpleUpdate.simple_update()` function?

Reply:
It basically works the same, pointing to tensor_network's attribute instead of a self attribute.